packages/fiori-elements-writer/src/types.ts

```diff
@@ -109,14 +110,23 @@ export interface FioriApp extends App {
 
 export interface FioriElementsApp<T> extends Ui5App {
     template: Template<T>;
-    service: Omit<OdataService, 'model'>; // Model name will use defaults
+    service: Omit<OdataService, 'model'> & {
+        capService?: CapServiceCdsInfo;
+    };
     app: FioriApp;
     appOptions: Partial<AppOptions> & {
         /**
          * Generate OPA based tests, if applicable to the specified template.
          * This will eventually move up to {@link Ui5App.appOptions}
          */
         addTests?: boolean;
+        /**
+         * Enables NPM workspaces for the application.
```

Comment:
I dont think this is strictly true. We dont enable npm workspaces for the application but for the a single top-level, root CAP package.